spec/lib/granite/routing/caching_spec.rb

```diff
@@ -32,14 +32,16 @@ def cache
       before { subject.clear_cache! }
 
       it 'returns instance of Cache' do
-        expect(subject.granite_cache).to be_kind_of Granite::Routing::Cache
+        expect(subject.granite_cache).to be_a Granite::Routing::Cache
       end
     end
   end
 
   describe '#clear_cache!' do
     it 'calls super' do
       allow_any_instance_of(super_module).to receive(:super)
+
+      expect { subject.clear_cache! }.not_to raise_error
```

Comment:
I think the intention of this spec is:
```suggestion
      expect_any_instance_of(super_module).to receive(:super)
      subject.clear_cache!
```

Reply:
Yup, it actually should even check for `clear_cache!` instead of `super`